uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -820,6 +843,25 @@ public void discoverCoreDeviceBroker(String brokerId) {
         log.info("Added IoT Core broker as {} with endpoint {}:{}", brokerId, endpoint, IOT_CORE_MQTT_PORT);
     }
 
+
+    /**
+     * Set up IoT core broker port.
+     *
+     * @param brokerId broker name in tests
+     * @param port broker port in tests
+     */
+    @And("I force to set broker {string} with port {int}")
```

Comment:
why is this needed?

Reply:
Why isn't this using discovery like our other tests? Our internal testing does not do this. It adds the proper connectivity information with localhost and the custom port.

Reply:
It that scenario we check case when client can connect to custom MQTT port AND can't connect to wrong port. For that we use discovery as usually but after that rewrite port in discovered list of  address information  to wrong port. In Evergreen similar thing did by other way.

Reply:
I'm looking at GGAD-1-T20 right now. It does not do this, it is changing the discovery information. What step do you claim is doing this?

Reply:
That inject invalid connectivity information for non-yet actual port number 9001 which resulting second client failed to connect on port 9001.

With OTF we have no step like ''add connectivity information with host "localhost" and port "9001" we use IPdetector instead and it can't send fake port to backend.

Reply:
And why don't you have step for that?

Reply:
We can try to implement  by reconfigure IPDetector but I expecting rice between components and scenario. 
We already got a lot of such issues and usually resolved it by adding steps for waiting until string appears in log file. With IPDetector it be hardly possible because IPDetector report action to log several times. Because we can stuck on other waiting step it hard to predict how many times IPDetector send information to backend and  how many entries about it will be logged.

Reply:
We just combine both methods.

Reply:
Please update this as suggested to set the connectivity info.

Reply:
Ok, Akezhan will continue on Monday due to national holidays.

Reply:
Fixed in latest commits